upup/pkg/fi/cloudup/awstasks/vpc.go

```diff
@@ -192,6 +197,10 @@ func (_ *VPC) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *VPC) error {
 		}
 	}
 
+	if len(changes.AdditionalCIDR) != 0 {
+		glog.Warningf("AdditionalCIDR changes on a VPC are not currently implemented")
```

Comment:
Not sure this makes sense to me.

> AdditionalCIDR changes on a VPC are not currently implemented

How about 

AdditionalCIDR allows for more than one CIDR in a kops configuration, but that CIDR cannot be used by kops as currently implemented.

Thoughts?  I know I am nit picking, but if we are putting in a warning, we want it to be as completely understandable as we can.

Reply:
To me, that sounds like the CIDR value is unsupported - i.e. I need to pick a different CIDR